eclipsePlugin/src/de/tobject/findbugs/builder/FindBugsWorker.java

```diff
@@ -367,8 +367,6 @@ private void updateBugCollection(Project findBugsProject, Reporter bugReporter,
             // props
             st.newPoint("storeBugCollection");
             FindbugsPlugin.storeBugCollection(project, resultCollection, monitor);
-        } catch (IOException e) {
-            FindbugsPlugin.getDefault().logException(e, "Error performing SpotBugs results update");
         } catch (CoreException e) {
             FindbugsPlugin.getDefault().logException(e, "Error performing SpotBugs results update");
         }
```

Comment:
The IOException was catched from the `FindbugsPlugin.storeBugCollection()` method, which does not throw this exception. Also, the two catch blocks are the same.